README.md

```diff
@@ -32,6 +32,9 @@ All the configuration is in the config.yml file, here it is:
 # Enable or disable Velocity Cool whitelist!
 enabled: true
 
+# Enable or disable autoupdate
+autoupdate: true
+
```

Comment:
i would change this to update checker, it isn't the true autoupdater

Reply:
fixed in [f6f168f](https://github.com/atikiNBTW/VelocityCoolList/pull/3/commits/f6f168f7c9ee74a15073924f43fb8fbcaee278bd)